wp-content/themes/core/components/share/share.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare( strict_types=1 );
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = \Tribe\Project\Templates\Components\share\Controller::factory( $args );
+?>
+<aside class="social-share">
+
+	<h6 class="social-share__title"><?php esc_html_e( 'Share This', 'tribe' );?></h6>
+
+	<ul class="social-share-networks" data-js="social-share-networks">
```

Comment:
Wasn't sure if you want this stuff added as component options, same with header and the aside wrapper.

Reply:
It's fine for now. We'll need to make a a11y pass later. We can address anything that needs args then.

Reply:
Yep, for starters we need to change `aside` to `div`, but can wait : )